homeassistant/components/openhome/manifest.json

```diff
@@ -2,6 +2,7 @@
   "domain": "openhome",
   "name": "Linn / OpenHome",
   "documentation": "https://www.home-assistant.io/integrations/openhome",
-  "requirements": ["openhomedevice==0.6.3"],
+  "requirements": ["openhomedevice==0.7.2"],
+  "dependencies": [],
```

Comment:
```suggestion
```

Reply:
Since I'm in `manifest.json` should I add

```json
  "ssdp": [
    {
      "st": "urn:av-openhome-org:service:Product:3"
    },
    {
      "st": "urn:av-openhome-org:service:Product:2"
    }
  ]
```

to reduce the requirement on netdisco? (https://github.com/home-assistant/netdisco/pull/232/files)

Reply:
That requires support in a config flow. So please do that in a separate PR. Thanks!

https://developers.home-assistant.io/docs/creating_integration_manifest/#ssdp